src/elements/content-sidebar/SidebarNav.js

```diff
@@ -77,7 +82,7 @@ const SidebarNav = ({
                             data-resin-target={SIDEBAR_NAV_TARGETS.BOXAI}
                             data-target-id="SidebarNavButton-boxAI"
                             data-testid="sidebarboxai"
-                            onClick={onPanelChange}
```

Comment:
Why this change was necessary?

Reply:
I felt that the Button's `onClick` should not really to be aware of the handler function and because I've changed the type of `onPanelChange` I didn't want to change Button's `onClick` type.